test/functional/functional_test_helper.js

```diff
@@ -112,6 +112,26 @@ export function resetPassword(token) {
   }
 }
 
+export function performSearch(context, query, callback) {
+  return function (done) {
+    apiUrl('/v2/search')
+      .then((url) => {
+        request
+          .get(`${url}?qs=${encodeURIComponent(query)}`)
+          .send({ authToken: context.authToken })
+          .end((err, res) => {
+            if (typeof callback !== 'undefined')
+              callback(res.body)
+
+            done(err, res)
+          })
+      })
+      .catch((e) => {
+        done(e)
+      })
+  }
+}
```

Comment:
Please check examples which use `fetch()` (via `postJson()`) further on and implement search-requests in a similar fashion.

This will simplify tests-code and guarantee proper error handling